indra/sources/indra_db_rest/client_api.py

```diff
@@ -428,12 +430,17 @@ def get_statements_for_paper(ids, ev_limit=10, best_first=True, tries=2,
     return stmts_from_json(stmts_json.values())
 
 
-def submit_curation(level, hash_val, tag, text, curator,
-                    source='indra_rest_client'):
+def submit_curation(hash_val, tag, text, curator, source='indra_rest_client',
+                    ev_hash=None, is_test=False):
```

Comment:
Let's add some docstrings to explain the arguments here, that would be really useful!